tests/services/file_delivery/format_deliver_files_service/test_format_delivery_files_service.py

```diff
@@ -0,0 +1,72 @@
+import mock
+
+from cg.services.file_delivery.fetch_file_service.models import (
+    DeliveryFiles,
+    SampleFile,
+    CaseFile,
+    DeliveryMetaData,
+)
+from cg.services.file_delivery.file_formatter_service.delivery_file_formatting_service import (
+    DeliveryFileFormattingService,
+)
+import pytest
+
+from cg.services.file_delivery.file_formatter_service.models import FormattedFiles, FormattedFile
+
+
+@pytest.mark.parametrize(
+    "formatter_service, formatted_case_files, formatted_sample_files, case_files, sample_files",
+    [
+        (
+            "generic_delivery_file_formatter",
+            "empty_case_files",
+            "expected_formatted_analysis_sample_files",
+            "empty_case_files",
+            "expected_moved_analysis_sample_delivery_files",
+        ),
+        (
+            "generic_delivery_file_formatter",
+            "expected_formatted_analysis_case_files",
+            "expected_formatted_analysis_sample_files",
+            "expected_moved_analysis_case_delivery_files",
+            "expected_moved_analysis_sample_delivery_files",
+        ),
+    ],
+)
```

Comment:
I think these kinds of heavily parametrized tests get hard to review and understand. Might just be flavour, though. Feel free to keep the tests as is.